app/src/main/java/com/globant/practice/presentation/view/adapter/SubscriberAdapter.java

```diff
@@ -120,6 +122,10 @@ public SubscriberViewHolder(View itemView) {
         public void blindSubscriberList(final User user) {
             nickname.setText(user.getLogin());
             this.user = user;
+            if (context != null && avatar != null) {
```

Comment:
I meant the url, not the context nor avatar, since it's being provided by the app. but the url is received from server side. What if it is null or empty?